core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -1759,6 +1763,7 @@ SDDkwd__(EXE_DIAGNOSTIC_EVENTS,		"OFF"),
  // exposure.
  DDkwd__(HBASE_DELETE_COSTING,		             "ON"),
  DDflt0_(HBASE_DOP_PARALLEL_SCANNER,             "0."),
+ DDkwd__(HBASE_ESTIMATE_ROW_COUNT_VIA_COPROCESSOR,   "OFF"),
```

Comment:
I like the co-processor approach because it encapsulates the HFile to region server code and enables SQL to use well defined and externalized  APIs of HBase. From this perspective, it should be good to remove the old estimate code entirely even with small performance penalty. This would help to maintain the code better with future releases of hbase